psydac/feec/global_projectors.py

```diff
@@ -279,7 +279,7 @@ def __call__(self, fun):
         Parameters
         ----------
         fun : callable or list/tuple of callables
-            Scalar components of the real-valued vector function to be
+            Scalar components of the complex-valued vector function to be
```

Comment:
Does the function need to be complex-valued? Or maybe you meant to write "of the real- or complex-valued vector function"?

Reply:
Exactly, the vector function could be complex-valued or real-valued. This commit should change it.
[c698724](https://github.com/pyccel/psydac/pull/302/commits/c698724f4dd37dce27dbc1a1ed22e774336daaf0)